thank-you.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Thank you</title>
+    <link rel="stylesheet" href="src/main.css">
+</head>
+<body>
+        <main> 
+                <section>
+                    <h1>
```

Comment:
This is not semantic. Wrapping an ```<h1>``` around ```p``` tags looks funky. If you want to manipulate the boldness and font size in css!